c/planarityApp/planarityHelp.c

```diff
@@ -28,55 +28,6 @@ char *GetProjectTitle()
            "\n==================================================\n";
 }
 
-/****************************************************************************
```

Comment:
Great catch noting that these don't really belong in the new `planarityHelp.c` file.